dockerfiles/Dockerfile

```diff
@@ -53,6 +53,7 @@ RUN echo "deb http://deb.debian.org/debian bullseye-backports main" >> /etc/apt/
     flex \
     bison \
     libyaml-dev \
+    libsnmp-dev \
```

Comment:
We presumably will also need this in the various package builds under [`packaging/distros`](https://github.com/fluent/fluent-bit/tree/master/packaging/distros) too.